### PR TITLE
Fix #159: Remove decode flag from calls to Append Error Algorithm

### LINKED ISSUE #159: Unecessary argument provided to Append Error Algorithm
https://w3c.github.io/media-source/index.html#sourcebuffer-append-error

Set 5 states:
"If decode error is true, then run the end of stream algorithm with the error parameter set to "decode"."

the Append Error Algorithm is always called with the decode error parameter set to true.

Step 5 should be removed and replace the nine instances stating "run the append error algorithm with the decode error parameter set to true" (every instance referring to the append error algorithm) with "run the append error algorithm"

### PR DESCRIPTION
**Pull request by jdsmith3000**

### CODE REVIEW

**Comment by jdsmith3000** [comment]
Per guidance from @paulbrucecotton, I'm staging this PR now, but it should not be merged until after CfC closes but BEFORE we send in the transition request.

File: media-source.js [hunk 1]
```diff
@@ -130,9 +130,6 @@
 
   function append_error_helper(doc, df, id, text) {
     link_helper(doc, df, id, 'append error algorithm');
```
Comment (wolenetz) [inline media-source.js]: With this helper changed, I'd expect to see more changes in the generated html.

**Comment by jdsmith3000** [comment]
I hope this is rebased now.  Please check.  I'm out this afternoon unfortunately, so if you can nudge this to the finish, please do.

**Comment by wolenetz** [comment]
Looking now.

File: media-source.js (hunk 1 above)
Comment (wolenetz) [inline media-source.js]: updated patch lgtm

**Comment by wolenetz** [comment]
LGTM. I'll merge this now.

**Pull request merged**